src/core/player.js

```diff
@@ -237,7 +237,7 @@ DM.provide('Player',
         switch (event.event)
         {
             case 'apiready': if (this.apiReady) return /* dispatch only once */; else this.apiReady = true; break;
-            case 'loadedmetadata': this.error = null; this.ended = false; break;
+            case 'start': this.error = null; this.ended = false; break;
```

Comment:
[Or](https://github.com/dailymotion/player/pull/931/files#diff-d16b257e5a1b6b41d98926c20bcd8f2eR8) we should keep  `loadedmetadata` here. Reseting `error` when we received a `loadedmetadata` event is useful, in particular when one of the quality doesn't work - but others do.